WalletWasabi/Services/Terminate/TerminateService.cs

```diff
@@ -99,12 +103,12 @@ private void Console_CancelKeyPress(object? sender, ConsoleCancelEventArgs e)
 		e.Cancel = true;
 
 		// ... instead signal back that the app should terminate.
-		SignalTerminate();
+		SignalTerminate(ctrlCPressed: true);
 	}
 
-	public void SignalTerminate()
+	public void SignalTerminate(bool ctrlCPressed)
```

Comment:
I believe RPC stop command should also bypass the shutdown prevention. I would remove this parameter and always set result to `true`. We could also rename function `SignalForceTerminate`